### ISSUE TITLE
Regarding the issue of blank figures produced after the update

### ISSUE THREAD

**Issue by pacmandoh** [sajuukLyu/ggunchull#3]
## Issue

Thanks for your excellent R pkg ❤️ !

However, I seem to have encountered an issue that I'm unsure whether it's isolated or widespread. After using your provided example code, I noticed blank figures being generated with the same number as the parameters within the `color` or `fill` groups. Upon examining the source code, it appears that the problem may lie with `plot.new()` in `step5`. I attempted to control the invocation of `plot.new()` during the loop process using `tryCatch()`, as well as directly calling `.External()` to generate data with `C_xspline`, but both attempts proved unsuccessful. As a novice, I humbly request your guidance in finding a solution.

**Note:** The `stat_unchull0` is functioning as intended.

## Info

<img width="1403" alt="SCR-20240427-gjrl" src="https://github.com/sajuukLyu/ggunchull/assets/133459587/43f757b1-5a55-4f61-852d-0ab3a0cb20ef">

```R
> R.version
               _                           
platform       aarch64-apple-darwin20      
arch           aarch64                     
os             darwin20                    
system         aarch64, darwin20           
status                                     
major          4                           
minor          3.3                         
year           2024                        
month          02                          
day            29                          
svn rev        86002                       
language       R                           
version.string R version 4.3.3 (2024-02-29)
nickname       Angel Food Cake
```

```R
> packageVersion("ggunchull")
[1] ‘1.0.0’
> packageVersion("graphics")
[1] ‘4.3.3’
> packageVersion("ggplot2")
[1] ‘3.5.1’
> packageVersion("alphahull")
[1] ‘2.5’
> packageVersion("dbscan")
[1] ‘1.1.12’
> packageVersion("polyclip")
[1] ‘1.10.6’
> packageVersion("magrittr")
[1] ‘2.0.3’
```

**Comment by sajuukLyu** [comment]
Sorry for being late. I think this issue should have been fixed in version 1.0.1. Thanks for pointing out the problem.

**Comment by pacmandoh** [comment]
@sajuukLyu thx!
